source/type/type.h

```diff
@@ -563,14 +563,30 @@ struct VehicleJourney: public Header, Nameable, hasVehicleProperties {
     std::vector<boost::shared_ptr<disruption::Impact>> get_impacts() const;
 
     bool operator<(const VehicleJourney& other) const;
-    template<class Archive> void serialize(Archive& ar, const unsigned int ) {
+
+    template<class Archive> void save(Archive& ar, const unsigned int ) const {
+        ar & name & uri & route & physical_mode & company & validity_patterns
```

Comment:
I don't think we can since the method is const in one case, and not const in the other one